extensions/hibernate-reactive/deployment/src/main/java/io/quarkus/hibernate/reactive/deployment/HibernateReactiveProcessor.java

```diff
@@ -297,7 +302,8 @@ private static ParsedPersistenceXmlDescriptor generateReactivePersistenceUnit(
             } else if (persistenceUnitConfig.sqlLoadScript.isPresent()) {
                 //raise exception if explicit file is not present (i.e. not the default)
                 throw new ConfigurationError(
-                        "Unable to find file referenced in '" + HibernateOrmProcessor.HIBERNATE_ORM_CONFIG_PREFIX
+                        "Unable to find file referenced in '"
+                                + HibernateOrmConfig.puPropertyKey(persistenceUnitConfigName, "sql-load-script") + "="
```

Comment:
The produced error message has duplicate typo:

> Unable to find file referenced in 'quarkus.hibernate-orm.sql-load-script=sql-load-script=import.sql'. Remove property or add file to your path.

cc @yrodiere @gsmet

Reply:
Thanks, here you go: https://github.com/quarkusio/quarkus/pull/19203